File: models/inventoryItem.py
import uuid

class InvetoryItem:
    def __init__(self, name, descript, rarity):
        self.itemId = uuid.uuid4()
        self.name = name
        self.descript = descript
        self.rarity = rarity   

    def __str__(self):
        return f"{self.name} ({self.rarity}) - {self.descript}"

File: INF122Assignment3/src/InventoryItem.java
import lombok.Getter;
import lombok.Setter;

import java.util.UUID;

@Getter
@Setter
public class InventoryItem {
    private UUID item_id;
    private String name;
    private String description;
    private String rarity;
    private int quantity;

    public InventoryItem(String name, String description, String rarity, int quantity) {
        this.item_id = UUID.randomUUID();
        this.name = name;
        this.description = description;
        this.rarity = rarity;
        this.quantity = quantity;
    }

    @Override
    public String toString(){
        return String.format("""
                \n%s (x%s)
                %s
                Rarity: %s""", name, quantity, description, rarity);
    }
}
